src/pages/noteForm/noteForm.css

```diff
@@ -0,0 +1,4 @@
+#myCanvas {
```

Comment:
This is not required I guess since you have not used it anywhere.

Reply:
Done.